File: API .NET/API .NET/Modele/Frais.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API_.NET.Modele
{
    public class Frais
    {
        public int id { get; set; }
        public int employe_id { get; set; }
        public string intitule { get; set; }
        public float montant { get; set; }
        public string devise { get; set; }
        public DateTime date { get; set; }
        public string statut { get; set; }
        public string motif { get; set; }
        public int note_frais_id { get; set; }

        public Frais()
        {
        }
        public Frais(int id, int employe_id, string intitule, float montant, string devise, DateTime date, int note_frais_id = 0, string statut = null, string motif = null)
        {
            this.id = id;
            this.employe_id = employe_id;
            this.intitule = intitule;
            this.montant = montant;
            this.devise = devise;
            this.date = date;
            this.statut = statut;
            this.motif = motif;
            this.note_frais_id = note_frais_id;
        }
        public override string ToString()
        {
            return $"{id} {employe_id} {intitule} {montant} {devise} {date} {statut} {motif} {note_frais_id}";
        }
    }
}

File: Appli_android/app/src/main/java/com/example/myapplication/models/Frais.kt
package com.example.myapplication.models

import com.google.gson.annotations.SerializedName
import java.sql.Date

data class Frais (
    @SerializedName("id")
    var id: Int,

    @SerializedName("employe_id")
    var employe_id: Int,

    @SerializedName("intitule")
    var intitule: String,

    @SerializedName("montant")
    var montant: Float,

    @SerializedName("devise")
    var devise: String,

    @SerializedName("date")
    var date: Date,

    @SerializedName("statut")
    var statut: String,

    @SerializedName("motif")
    var motif: String,

    @SerializedName("note_frais_id")
    var note_frais_id: Int
    )